source/fundamentals/builders/projections.txt

```diff
@@ -11,3 +11,169 @@ Projections Builders
    :class: singlecol
 
 .. _projections-builders:
+
+Overview
+--------
+
+MongoDB supports **field projection**, specifying which fields to include and exclude when returning results from a
+query. Projection in MongoDB follows some basic rules:
+
+- the :guilabel:`_id` field is **always** included and must explicitly be excluded
+- specifying a field for inclusion implicitly excludes all other fields **except** the :guilabel:`_id` field
+- specifying a field for exclusion removes only that field in a query result
+
+Find more information :manual:`here </tutorial/project-fields-from-query-results/>`.
```

Comment:
For more information on which part? Could you say something like "For more information on selecting projection fields from query results, see ``?